docs/sql-migration-guide.md

```diff
@@ -77,6 +77,8 @@ license: |
 
   - In Spark 3.2, `CREATE TABLE .. LIKE ..` command can not use reserved properties. You need their specific clauses to specify them, for example, `CREATE TABLE test1 LIKE test LOCATION 'some path'`. You can set `spark.sql.legacy.notReserveProperties` to `true` to ignore the `ParseException`, in this case, these properties will be silently removed, for example: `TBLPROPERTIES('owner'='yao')` will have no effect. In Spark version 3.1 and below, the reserved properties can be used in `CREATE TABLE .. LIKE ..` command but have no side effects, for example, `TBLPROPERTIES('location'='/tmp')` does not change the location of the table but only create a headless property just like `'a'='b'`.
 
+  - In Spark 3.2, `TRANSFORM` operator can't support alias in inputs. In Spark 3.1 and earlier, we can write script transform like `SELECT TRANSFORM(a AS c1, b AS c2) USING 'cat' FROM TBL`.
+
```

Comment:
Is there any valid use case around it? Can users access `c1, c2` somehow? e.g. `SELECT c1 FROM (SELECT TRANSFORM ...)`

Reply:
> Is there any valid use case around it? Can users access `c1, c2` somehow? e.g. `SELECT c1 FROM (SELECT TRANSFORM ...)`

```
SELECT TRANSFORM(a as a1, sum(b) as sum_b)
USING 'cat'
FROM tbl
HAVING sum_b > 10
```

Reply:
this looks super weird and not SQL-ish.